src/test/scala/com/cognite/sdk/scala/v1/SequenceRowsTest.scala

```diff
@@ -6,9 +6,8 @@ package com.cognite.sdk.scala.v1
 import cats.data.NonEmptyList
 import com.cognite.sdk.scala.common._
 import io.circe.syntax._
-import org.scalatest.ParallelTestExecution
 
-class SequenceRowsTest extends SdkTestSpec with ParallelTestExecution with RetryWhile {
+class SequenceRowsTest extends SdkTestSpec with RetryWhile {
```

Comment:
Looks like solved!  Thanks a lot, @wjoel :rocket: Should we merge after reverting this?

Reply:
Yes, that wasn't the culprit, so lets revert it and merge.